content/en/posts/outer-misalignment.md

```diff
@@ -0,0 +1,11 @@
+Intermediate
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you unintentionally teach it to fetch any paper-related item, including your important documents. This is a simple analogy for outer misalignment – the dog's action doesn't align with your true intentions.
```

Comment:
I have to think about this some more and get clarity myself, but it sounds more like inner misalignment to me. The dog being confronted with other types of important documents sounds like an out of distribution event.

I'll get back to this. I might be confused.

Reply:
This point is still a little unclear to me. Do you feel you have clarity on this? i.e. do you feel confident that this is an example of outer misalignment? If yes, we can merge it and if necessary amend it later. Can you give me an example of the difference of outer and inner misalignment e.g. with the dog?

Reply:
You are correct, good catch! The issue is the word 'unintentionally', which actually implies that this is an example of inner misalignment. If the dog was explicitly told to fetch all paper-related items, that would be outer misalignment. If the dog was told to fetch newspapers but misinterpreted that as all paper-related items, that would be inner misalignment. I'll try to edit it, but as I said on Signal earlier, I'm having problems with Github, especially when it comes to editing posts

Reply:
I believe I've now edited the post

Reply:
Upon further reflection, I think the dog example is actually still a bit difficult - the problem here is that when training a dog, there isn't really this middle step where inner and outer misalignment would be differentiated. So the dog example could in a way be an example of both, and thus doesn't really help making the distinction between the two concepts clear.

Reply:
I changed the phrasing a bit now to make clearer that it's really about mistakes made by you as the trainer, rather than misgeneralized goals due to a distribution shift.